### ISSUE TITLE
Display calendar 

### ISSUE THREAD

**Issue by Leticia-maria** [geem-lab/seminars#11]
Some ideas to implement here:

- [x] display calendar
- [ ] link to the google calendar (maybe sending an invite.ics (?) to registered emails)

**Comment by schneiderfelipe** [comment]
[This might be useful](https://github.com/ics-py/ics-py).

**Comment by Leticia-maria** [comment]
Almost there, @schneiderfelipe :
- Dependencies:
https://github.com/geem-lab/seminars/blob/6fe5a3b79865a515b6573c7fd886b5390336e5d5/scripts/update-seminars.py#L6-L7

- Calendar (I think it is better to do all in one line, in this case):
https://github.com/geem-lab/seminars/blob/6fe5a3b79865a515b6573c7fd886b5390336e5d5/scripts/update-seminars.py#L160-L178

I have already updated poetry.lock and the TOML file

**Comment by Leticia-maria** [comment]
Thank you for your comments, @schneiderfelipe. 

1. I have solved the dependency with IPython (the calendar is being displayed now by using ```markdown```, since it is an HTML code): 
- 21f7c9585f120402414ce23749c14bddb4ba2228

(I have used this solution because it is quite simple, and the design is according to the rest of the web page)
2. Today's date is highlighted:
https://github.com/geem-lab/seminars/blob/9fcc17cc136e1385f551185668658c76619403a4/scripts/update-seminars.py#L170-L178
